src/mlpack/core/util/mlpack_main.hpp

```diff
@@ -332,6 +332,9 @@ PARAM_FLAG("verbose", "Display informational messages and the full list of "
 
 #elif(BINDING_TYPE == BINDING_TYPE_R) // This is a R binding.
 
+// This doesn't actually matter for this binding type.
+#define BINDING_MATRIX_TRANSPOSED true
```

Comment:
We might want to think about this one again later.  The binding it really matters for is `nmf`.  For NMF, given a matrix V, we want to decompose it into `V ~= W * H`.  So, we need to make sure that we can just take whatever `W` and `H` we get from the mlpack NMF binding in R and multiply them together.  Anyway, there is no huge need to discuss now, but let's just try and remember that as we get closer to having everything working, we should hand-test the NMF binding from R to make sure that `true` is the right value here. :+1:

Reply:
Yeah, This sounds good.